Assets/MixedRealityToolkit.SDK/Profiles/DefaultMixedRealitySpeechCommandsProfile.asset

```diff
@@ -27,22 +28,9 @@ MonoBehaviour:
       id: 1
       description: Toggle Diagnostics
       axisConstraint: 0
-  - keyword: Toggle Frame Rate
-    keyCode: 102
-    action:
-      id: 2
-      description: Toggle Frame Rate
```

Comment:
Should the old Diagnostics actions be removed from the actions profile? (or are they, and I missed it?)

Reply:
i'll double check.

Reply:
there weren't any custom actions